README.md

```diff
@@ -1,24 +1,48 @@
- # Fullstack Boilerplate
+ # Fullstack Boilerplate Express/Typescript/React
 
-Use it for setting up a take-home interview assignment or a portfolio project.
+A backbone for your coding challenge.
 
 ## Contents
 
-- [Backend service](https://github.com/DevSkillsHQ/fullstack-boilerplate-express-react/tree/main/app-express) - an Express service with a `/ping` endpoint. Extend with your code.
-- [Frontend app](https://github.com/DevSkillsHQ/fullstack-boilerplate-express-react/tree/main/app-react) - a React app. Extend with your code.
-- [E2E test suites](https://github.com/DevSkillsHQ/fullstack-boilerplate-express-react/tree/main/cypress/integration) - a backend and a frontend Cypress test suites. Extend with your tests.
-- [Pipeline](https://github.com/DevSkillsHQ/fullstack-boilerplate-express-react/blob/main/.github/workflows/tests.yml) - a test Runner that executes the Cypress tests on push to a branch other than `master`/`main`.
+- [Backend service](app-express) - an Express service with a `/ping` endpoint. Extend with your code.
+- [Frontend app](app-react) - a React app. Extend with your code.
+- [E2E test suites](cypress/e2e) - a backend and a frontend Cypress test suites. Extend with your tests.
 
 ## Tech Stack
 
-- React
+- React 18
```

Comment:
@Ostter Can dependabot do major lib upgrades? If so, specifying the version in the README might get obsolete.

Reply:
it can
worth to remove versions of all libraries from the readme